LICENSE

```diff
@@ -353,12 +353,6 @@ Copyright 2014 - 2016 Real Logic Ltd.
 
 ---------------
 
-pekko-protobuf contains the sources of Google protobuf 2.5.0 runtime support,
```

Comment:
Do we already have a notice that we publish the v3 versions? If not, maybe we can repurpose this one.

Reply:
We don't have the sources of protobuf-java v3 copied into our code base. With protobuf-java v3, sbt-assembly is used to take the official google jar and to produce a new pekko-protobuf-v3 jar with shaded package names. The LICENSE file accounts for source code. We have the `legal` folder where we have specific LICENSE files that we add to our jars. There is one of those for pekko-protobuf-v3 jar that acknowledges that the classes are derived from Google classes.

Reply:
Thanks for clarifying, makes sense.